src/inference/api_server.py

```
@@ -1706,6 +1707,19 @@ async def _no_graph():
+    # 기존 interrupt 상태가 남아있으면 거절(cancel)로 해소
+    try:
+        from langgraph.types import Command
+
+        existing_state = await manager.graph.aget_state(config)
+        if existing_state and existing_state.next:
+            await manager.graph.ainvoke(
+                Command(resume={"approved": False, "cancel": True}),
+                config,
+            )
+    except Exception:
+        pass  # 상태 확인 실패는 무시하고 새 실행 진행
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)